src-tauri/tauri.conf.json

```
@@ -33,6 +38,15 @@
                         "args": true
                     },
                     {
+                        "name": "amass",
+                        "cmd": "amass",
+                        "args": true
+                    },
+                    {
+                        "name": "snmp-check",
+                        "cmd": "snmp-check",
+                        "args": true
+                    },
                         "name": "arjun",
```

DDT is broken because arjun is missing its opening bracket. 
```suggestion
{
                        "name": "arjun",
```